src/commands/sites/sites-create-template.js

```diff
@@ -0,0 +1,218 @@
+// @ts-check
+
+const inquirer = require('inquirer')
+const pick = require('lodash/pick')
+const prettyjson = require('prettyjson')
+
+const { chalk, error, getRepoData, log, logJson, track, warn } = require('../../utils')
+const { configureRepo } = require('../../utils/init/config')
+const { getGitHubToken } = require('../../utils/init/config-github')
+const { createRepo, getTemplatesFromGitHub } = require('../../utils/sites/utils')
+
+const { getSiteNameInput } = require('./sites-create')
+
+let ghToken
+
+const fetchTemplates = async (token) => {
+  const templatesFromGithubOrg = await getTemplatesFromGitHub(token)
+
+  return templatesFromGithubOrg
+    .filter((repo) => !repo.archived && !repo.private && !repo.disabled)
+    .map((template) => ({
+      name: template.name,
+      sourceCodeUrl: template.html_url,
+      slug: template.full_name,
+    }))
+}
+
+/**
+ * The sites:create-template command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const sitesCreateTemplate = async (options, command) => {
+  const { api } = command.netlify
+
+  await command.authenticate()
+
+  const { globalConfig } = command.netlify
+  ghToken = await getGitHubToken({ globalConfig })
```

Comment:
```suggestion
  const ghToken = await getGitHubToken({ globalConfig })
```

[sand] I don't think we need `ghToken` at the top level